src/js/components/reviewData/CertificationModal/ReviewDataCertifyModal.jsx

```diff
@@ -47,19 +49,16 @@ export default class ReviewDataCertifyModal extends React.Component {
 	clickedCertifyButton(e) {
 		e.preventDefault();
 
-		// show the progress bar
-		this.setState({
-			showProgress: true,
-			closeable: false
-		}, () => {
-			// mock the API completion
-			window.setTimeout(() => {
-				this.setState({
-					publishComplete: true,
-					closeable: true
-				});
-			}, 5 * 1000);
-		});
+		ReviewHelper.certifySubmission(this.props.submissionID)
+			.then(() => {
+				this.closeModal();
+			})
+	        .catch((error) => {
+	        	console.log(error)
+	            this.setState({
+	            	errorMessage: "An error occurred while attempting to certify the submission. Please contact your administrator for assistance."
```

Comment:
Here we probably want to change the error message if it's a 401 or 403 HTTP status code